Common/UnitDefinitions/RelativeHumidity.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Name": "RelativeHumidity",
+  "BaseUnit": "Percent",
+  "XmlDoc": "Relative humidity is a ratio of the actual water vapor present in the air to the maximum water vapor in the air at the given temeperature.",
+  "Units": [
+    {
+      "SingularName": "Percent",
```

Comment:
You mentioned adding Fraction. Will you not be adding that?

Reply:
I've considered that initially but I'm not 100% it will be that useful. I think might make sense to wait until people start complaining about lack of it. ("0.3 RH" didn't make much sense to me so it would probably be just confusing if someone tries to serialize it)